src/webextension/manifest.json.tpl

```diff
@@ -5,6 +5,13 @@
 
   "description": "{{description}}",
 
+  "applications": {
+    "gecko": {
+      "id": "lockbox@mozilla.com",
+      "update_url": "https://testpilot.firefox.com/files/lockbox@mozilla.com/updates.json"
```

Comment:
Is this the right place to put update URLs? I would have expected it to go in our `install.rdf`, since that's the "root" extension. I have no idea what's right here though, so if you've confirmed that this is ok with the Test Pilot folks, then this looks good to me.

Reply:
@jimporter Good catch and fair question. I poked around and saw other Test Pilot experiments with emebedded WebExtensions (including [containers](https://github.com/mozilla/testpilot-containers/blob/master/webextension/manifest.json#L15)) using the manifest.json for the updated URL. Either way, I've added a reminder to test and confirm this.

Reply:
This is correct (assuming your ID is correct)